eduiddashboard/utils.py

```diff
@@ -191,3 +194,195 @@ def convert_to_localtime(dt):
     dt = dt.replace(tzinfo=pytz.utc)
     dt = dt.astimezone(tz)
     return dt
+
+
+def sanitize_get(request, *args):
```

Comment:
Is it really the best way to have four almost identical wrappers around sanitize_input here?

It seems like there would be only one additional argument to the function (content_type) if they are collapsed into one function?